tests/page-templates/19-mobile/02-mobile-connection-disabled.js

```diff
@@ -0,0 +1,23 @@
+/*eslint-env mocha*/
+/*global BOOMR_test,assert*/
+
+describe("e2e/19-mobile/02-mobile-connection-disabled", function() {
+	var t = BOOMR_test;
+	var tf = BOOMR.plugins.TestFramework;
+
+	it("Should pass basic beacon validation", function(done) {
+		t.validateBeaconWasSent(done);
+	});
+
+	it("Should have no keys starting with mob", function() {
+		var k, actual = true, b = tf.beacons[0];
+
+		for (k in b) {
+			if (k.indexOf("mob") === 0) {
+				actual = false;
+			}
+		}
+
+		assert.isTrue(actual);
```

Comment:
This test fails. I left it here because I would like to still be able to do it. How can you disable the mobile plugin for testing? I noticed that `BOOMR.plugins` does not have a `mobile` property. The plugin is available and working, but `BOOMR` is not aware of it and it does not allow me to disable it like I normally would.

Reply:
This no longer fails! With the plugin refactor, it can be disabled and the test does what it should.